### DISCUSSION TITLE
Issue Running Ktlint for Oppia Android: "Unable to Access Jarfile" Error

### DISCUSSION THREAD

**Discussion by TanishMoral11** [oppia/oppia-android#5585] [Q&A (Other)] [answered]
Hello everyone,  

I have successfully set up Oppia Android on my system. While attempting to fix lint failures using `ktlint`, I ran the following command from the terminal:  

```
java -jar /home/runner/oppia-android-tools/ktlint -F --android domain/src//.kt utility/src//.kt data/src//.kt app/src//.kt testing/src//.kt scripts/src//.kt instrumentation/src//*.kt
```
However, I encountered the following error:
![image](https://github.com/user-attachments/assets/15fc81bd-c8e6-47fe-a2ed-4d9ccb81aa7e)

Error: Unable to access jarfile /home/runner/oppia-android-tools/ktlint

It seems like the ktlint jar file is either missing or not accessible. I would appreciate guidance on resolving this issue.

Specifically, I would like to know:

How to ensure the ktlint jar file is correctly set up in the /home/runner/oppia-android-tools/ directory.
Whether there are any additional steps to configure or download ktlint for Oppia Android.
Thank you

**Comment by adhiamboperes** [comment, marked as answer]
Hi @TanishMoral11 cc/@theayushyadav11, this typically happens if you missed setting up ktlint during the project setup. 

As per [this](https://github.com/oppia/oppia-android/wiki/Installing-Oppia-Android#install-oppia-android) Wiki section, you need to run `bash scripts/setup.sh` in your terminal at the project root to set up a number or linting tools, including **Checkstyle**, **Buf** and **Buildifier**. 

![image](https://github.com/user-attachments/assets/c3b8e524-f2ee-4ae2-afa5-cca48ae848aa)

Please mark as answer if this works for you.

**Reply by TanishMoral11** [reply]
Thank You So Much @adhiamboperes !!
